docs/contributing.rst

```diff
@@ -92,7 +92,7 @@ Run docker::
 
 To run the tests, use::
 
-    ./manage.py test resolwe_bio
+    ./manage.py test resolwe_bio --parallel=2
```

Comment:
Either also enable parallel tests in Tox or omit this change (and make it in a separate commit that enables parallel tests).